assets/dev/js/editor/utils/container-helper.js

```diff
@@ -53,6 +54,25 @@ export class ContainerHelper {
 		} );
 	}
 
+	static createGridContainer( settings = {}, target, options = {} ) {
```

Comment:
@ArmanPurtoian I don't think we need a new function for this? It looks identical to `createContainer()`. You just need to pass the correct `settings` to `createContainer()`, no?